README.md

```diff
@@ -61,6 +61,17 @@ Host github.com
 
 Note: parallel downloads is only available for repositories and not forge modules.
 
+### Parallel tests
+It is also possible to use the `parallel_tests` Gem via the `:parallel_spec` task.  Use of parallelization at this level can result in large performance benefits when a system has a number of CPUs and the Rspec examples involve many full catalog compilations, such as in a control repo that compiles catalogs for every host for example.
```

Comment:
Good idea, a caveat to say "This is most efficent with modules with a large number of spec files, rather than spec test count"

There is a way of bucketing all the tests, and running them in parralel so it doesn't matter how many files there are, but it's a lot harder because of dependancy management and making sure all the tests can run in parralel...